pkg/machines/README.md

```diff
@@ -16,7 +16,7 @@ In the vagrant:
         virt-install -r 128 --pxe --force --nographics --noautoconsole \
         -f ${NAME}.img -n ${NAME}   # to create and start a dummy VM
 
-Log into Cockpit as the 'root' user (pwd 'foobar') and follow `Virtual Machines` from the left-side menu.
+Log into Cockpit as the 'root' user (pwd 'foobar') and follow `Virtual machines` from the left-side menu.
```

Comment:
Kept as Title Case, if we adopt the navigation as title case PF4 recommendation.